controller/space.go

```diff
@@ -330,22 +348,40 @@ func ConvertSpaceToModel(appSpace app.Space) space.Space {
 type SpaceConvertFunc func(*goa.RequestData, *space.Space, *app.Space)
 
 // ConvertSpacesFromModel converts between internal and external REST representation
-func ConvertSpacesFromModel(request *goa.RequestData, spaces []space.Space, additional ...SpaceConvertFunc) []*app.Space {
+func ConvertSpacesFromModel(ctx context.Context, db application.DB, request *goa.RequestData, spaces []space.Space, additional ...SpaceConvertFunc) ([]*app.Space, error) {
 	var ps = []*app.Space{}
 	for _, p := range spaces {
-		ps = append(ps, ConvertSpaceFromModel(request, p, additional...))
+		spaceData, err := ConvertSpaceFromModel(ctx, db, request, p, additional...)
+		if err != nil {
+			return nil, err
+		}
+
+		ps = append(ps, spaceData)
 	}
-	return ps
+	return ps, nil
 }
 
 // ConvertSpaceFromModel converts between internal and external REST representation
-func ConvertSpaceFromModel(request *goa.RequestData, p space.Space, additional ...SpaceConvertFunc) *app.Space {
+func ConvertSpaceFromModel(ctx context.Context, db application.DB, request *goa.RequestData, p space.Space, additional ...SpaceConvertFunc) (*app.Space, error) {
 	selfURL := rest.AbsoluteURL(request, app.SpaceHref(p.ID))
 	relatedIterationList := rest.AbsoluteURL(request, fmt.Sprintf("/api/spaces/%s/iterations", p.ID.String()))
 	relatedAreaList := rest.AbsoluteURL(request, fmt.Sprintf("/api/spaces/%s/areas", p.ID.String()))
 	relatedBacklogList := rest.AbsoluteURL(request, fmt.Sprintf("/api/spaces/%s/backlog", p.ID.String()))
 	relatedCodebasesList := rest.AbsoluteURL(request, fmt.Sprintf("/api/spaces/%s/codebases", p.ID.String()))
 
+	var backlogCounter int
+	err := application.Transactional(db, func(appl application.Application) error {
+		backlogItems, err := appl.WorkItemTypes().ListPlannerItems(ctx, p.ID)
```

Comment:
This is showing the number of WITs that extend plannerItem? 

What we're interested in there are the total number of WIs in the backlog.